common/data/src/main/java/org/thingsboard/server/common/data/StringUtils.java

```diff
@@ -245,4 +247,27 @@ public static String truncate(String string, int maxLength, Function<Integer, St
         return string.substring(0, maxLength) + truncationMarkerFunc.apply(truncatedSymbols);
     }
 
+    public static List<String> getListValuesWithoutQuote(String value) {
```

Comment:
As we discussed, please change the name to something like `splitByCommaWithoutQuotes`.